ontology/yaml/resources/SAFETY/entity_types/EHT.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+########################
+### Canonical Types  ###
+########################
+
+
+EHT_SS:
```

Comment:
Can you please clarify what you mean by "heat tracer" - technically, this is a piece of equipment used to heat piping in HVAC. If you are trying to model a sensor or something that detects heat, I would instead create a new field like `heat_detection_status`.

Reply:
I was told to put it in SAFETY since a heat trace prevents pipe to freeze. It has just a start/stop status and no sensor.

![image](https://github.com/google/digitalbuildings/assets/21256796/6dc99b24-2542-403c-9a91-1b9a95b3782b)

Reply:
Thank you for clarifying